geocoder_tester/world/france/aquitaine/test_addresses.csv

```diff
@@ -515,6 +515,6 @@ query,lon,lat,limit,expected_name,expected_housenumber,expected_street,expected_
 318 Chemin Bordenave Urcuit,,,,318 Chemin Bordenave,,,,64990
 318 Chemin Bordenave 64990,,,,318 Chemin Bordenave,,,Urcuit,
 318 Chemin Bordenave,-1.356450,43.495035,,318 Chemin Bordenave,,,,64990
-16 Rue de Hiribehere Ustaritz,,,,16 Rue de Hiribehere,,,,64480
-16 Rue de Hiribehere 64480,,,,16 Rue de Hiribehere,,,Ustaritz,
-16 Rue de Hiribehere,-1.457295,43.400521,,16 Rue de Hiribehere,,,,64480
+16 Rue de Hiribehere Ustaritz,,,,16 Rue Hiribehere,,,,64480

```

Comment:
You sure about that one?
Fantoir says like you http://cadastre.openstreetmap.fr/fantoir/#insee=64547&tab=1 but the street sign do has the `de`:

![screenshot from 2016-10-31 15-30-21](https://cloud.githubusercontent.com/assets/146023/19857718/ffc79aec-9f7e-11e6-96d5-316bc1c1d44c.png)